Overview.src.html

```diff
@@ -934,6 +934,41 @@ <h4 id=terminology-headers>Headers</h4>
 Unless stated otherwise, it is unset.
 
 <p>A <span title=concept-request>request</span> has an associated
+<dfn title=concept-request-use-token-binding>use-token-binding flag</dfn>.
+Unless stated otherwise, it is unset.
+<p class="note no-backref"> <span title=concept-request>Request</span>'s
+<span title=concept-request-use-token-binding>use-token-binding</span> flag indicates whether the
+user agent will send its <span title=concept-token-binding-id>Token Binding Id</span>
+for the <span title=concept-request-origin>origin</span> along with the
+<span title=concept-request>request</span> when it transmits the
+<span title=concept-request>request</span> to the server.
+A <dfn title=concept-token-binding>Token Binding</dfn> is a public-private key-pair created by a
```

Comment:
This and the definition of "Token Binding Id" should probably be moved into a new "Infrastructure" subsection. Defining things in a note doesn't make much sense.